src/main/java/frc/robot/commands/autonomous/FiveCycle.java

```diff
@@ -89,10 +89,14 @@ public Command autoHelper(PathPlannerPath pathOne, PathPlannerPath pathTwo) {
         AutoBuilder.followPath(pathOne)
             .deadlineFor(
                 Commands.sequence(
+                    robotContainer
```

Comment:
20 milliseconds is very short, basically one time through the loop.
prepScoreCoral does a bunch of things.   That short of a timeout could have problems